### PR TITLE
Add multi-tenancy subproject to sig-auth

### PR DESCRIPTION
**Pull request by nikhita**
Ref: https://github.com/kubernetes/org/issues/528

/cc @liggitt @tallclair @mikedanese @easeway @davidopp

### CODE REVIEW

File: sigs.yaml [hunk 1]
```diff
@@ -437,6 +437,11 @@ sigs:
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/kubelet/token/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/serviceaccount/OWNERS
       - https://raw.githubusercontent.com/kubernetes/kubernetes/master/plugin/pkg/admission/serviceaccount/OWNERS
+    - name: multi-tenancy
+      description: >
+       Multi-tenancy related proposals and prototypes.
```
Comment (tallclair) [inline sigs.yaml]: Can you be a bit more specific here? For instance, adding a new authorization feature that enables multitenant usecases would fall under the authorizers subproject, even if it's "multi-tenancy related".

Comment (nikhita) [reply sigs.yaml]: I'll leave it to @easeway and @davidopp to come up with a proper description. :)

Comment (easeway) [reply sigs.yaml]: My proposal is: _proposals and prototypes for introducing tenant model to enable multi-tenant cluster_

Comment (nikhita) [reply sigs.yaml]: Done!

**Comment by tallclair** [comment]
/lgtm
/approve

We can always iterate on the description as the project evolves.

**Comment by nikhita** [comment]
/assign @idvoretskyi 
for approval

**Pull request merged**